src/main/java/de/tum/in/www1/artemis/service/CourseService.java

```diff
@@ -703,11 +703,22 @@ public ZonedDateTime determineEndDateForActiveStudents(Course course) {
     public int determineTimeSpanSizeForActiveStudents(Course course, ZonedDateTime endDate, int maximalSize) {
         var spanTime = maximalSize;
         if (course.getStartDate() != null) {
-            var mondayInWeekOfStart = course.getStartDate().with(DayOfWeek.MONDAY).withHour(0).withMinute(0).withSecond(0).withNano(0);
-            var mondayInWeekOfEnd = endDate.plusWeeks(1).with(DayOfWeek.MONDAY).withHour(0).withMinute(0).withSecond(0).withNano(0);
-            var amountOfWeeksBetween = mondayInWeekOfStart.until(mondayInWeekOfEnd, ChronoUnit.WEEKS);
+            var amountOfWeeksBetween = calculateWeeksBetweenDates(course.getStartDate(), endDate);
```

Comment:
```suggestion
            long amountOfWeeksBetween = calculateWeeksBetweenDates(course.getStartDate(), endDate);
```
I think the code style suggests not using `var` for basic types.

(Nitpick, not introduced in this PR: It should be `numberOfWeeksBetween` since they are countable.)